tests/Feature/Facades/SpApiCredentialProviderFacadeTest.php

```diff
@@ -0,0 +1,27 @@
+<?php
+
+namespace Tests\Feature\Facades;
+
+use Glue\SpApi\Laravel\Facades\SpApiCredentialProvider;
+use Tests\TestCase;
+
+class SpApiCredentialProviderFacadeTest extends TestCase
+{
+    public function test_it_uses_same_instance_on_each_static_call()
+    {
+        $spApiCredentialProvider1 = SpApiCredentialProvider::getFacadeRoot();
+        $spApiCredentialProvider2 = SpApiCredentialProvider::getFacadeRoot();
+
+        $this->assertSame($spApiCredentialProvider1, $spApiCredentialProvider2);
+    }
+
+    public function test_it_uses_same_instance_when_facade_is_mocked()
+    {
+        SpApiCredentialProvider::shouldReceive()->zeroOrMoreTimes();
+        $spApiCredentialProvider1 = SpApiCredentialProvider::getFacadeRoot();
+        SpApiCredentialProvider::shouldReceive()->zeroOrMoreTimes();
+        $spApiCredentialProvider2 = SpApiCredentialProvider::getFacadeRoot();
+
+        $this->assertSame($spApiCredentialProvider1, $spApiCredentialProvider2);
+    }
```

Comment:
Ok so here, since this is the `SpApiCredentialProvider` facade it will be the same instance when you call `getFacadeRoot()` statically, because you don't have the `use RefreshesFacadeInstance;` on the `SpApiCredentialProvider` class?